### PR TITLE
Added feature template

### PR DESCRIPTION
**Pull request by RishiKumarRay**
Added Feature Template as asked.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: Feature Request/Enhancement
+description: Have a new idea of any feature you want to propose ? Please suggest.
```
Comment (kirillbobyrev) [inline .github/ISSUE_TEMPLATE/feature.yml:2]:
```suggestion
description: Suggest an idea for this project
```

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    attributes:
+      label: Additional information
+      description: Is there any additional information you wish to add?
+    validations:
+      required: false
+  - type: markdown
+    attributes:
+      value: |
+        You can Summit your Request.
+
```
Comment (kirillbobyrev) [inline .github/ISSUE_TEMPLATE/feature.yml:38]: I think, most of this is not really relevant, it'd be safe to omit these fields.

```suggestion
```

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+title: '[FEATURE] <description>'
+labels: ['goal: Addition']
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A brief description of the enhancement or Feature you want propose.
+    validations:
+      required: true
```
Comment (kirillbobyrev) [inline .github/ISSUE_TEMPLATE/feature.yml:12]:
```suggestion
  - type: textarea
    attributes:
      description: A clear description of the feature you would like to see.
      placeholder: |
        What problem does this solve? How does this feature improve the engine?
    validations:
      required: true
  - type: textarea
    attributes:
      description: Context
      placeholder: |
        Is this feature supported in other engines (e.g. Stockfish, Leela Chess Zero)?
        What is the rationale behind adding this enhancement?
    validations:
      required: true
```

**Review by kirillbobyrev — COMMENTED** [review]
Hi again! Thanks a lot, this is really useful! I've added some suggestions/clarifications to make the form smaller (and therefore reducing the entry barrier).

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  - type: textarea
+    attributes:
+      description: Context
+      placeholder: |
+        Is this feature supported in other engines (e.g. Stockfish, Leela Chess Zero)?
+        What is the rationale behind adding this enhancement?
+    validations:
+      required: true
```
Comment (kirillbobyrev) [inline .github/ISSUE_TEMPLATE/feature.yml:20]:
```suggestion
      required: false
```

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+name: Feature Request
+description: Suggest an idea for this project
+title: '[Feature]: '
+labels: ['feature']
+body:
+  - type: textarea
+    attributes:
+      description: A clear description of the feature you would like to see.
```
Comment (kirillbobyrev) [inline .github/ISSUE_TEMPLATE/feature.yml:8]:
```suggestion
      description: A clear description of the feature you would like to have.
```

**Review by kirillbobyrev — COMMENTED** [review]
Thanks a lot, Looks Good To Me now, just a couple of nits and I will merge the PR.

Also, feel free to pick up any other issues if you'd like to work on these: feel free to ask for specific things you would be interested to work on. I'm currently working on many features and they will gradually appear in the repo. Hopefully, I'll start creating/marking some issues as `good first issue` and `help wanted` relatively soon.

**Review by kirillbobyrev — APPROVED** [review]

**Comment by RishiKumarRay** [comment]
@kirillbobyrev Done with these changes too

**Pull request merged**